frontend/app/views/notes/_template.html.erb

```diff
@@ -36,31 +36,12 @@
   </div>
 <% end %>
 
-
-
-<div id="template_note_summary"><!--
```

Comment:
Correction: I've moved it to shared/templates so the js serialisation of templates doesn't freak out when with the comments used by this template.